pkg/client/devops/jenkins/devops.go

```diff
@@ -15,10 +15,17 @@ package jenkins
 
 import (
 	"kubesphere.io/devops/pkg/client/devops"
+	"net/http"
 )
 
 func NewDevopsClient(options *Options) (devops.Interface, error) {
-	jenkins := CreateJenkins(nil, options.Host, options.MaxConnections, options.Username, options.Password)
+	// we have to create http client with no redirection
```

Comment:
I don't understand why we need to have the no redirection client.

Reply:
If we make http request with POST method to some Jenkins endpoint, Jenkins will result in 302 back to http client, and http client will redirect to `Location`. If we don't specific `Referer` header, the http client will request same API using GET method again. That's not what we expected.